src/items.rs

```diff
@@ -5,7 +5,7 @@ mod machine_gun;
 mod muscet;
 mod sproinger;
 mod sword;
-
+mod turtle_shell;
```

Comment:
Looks like turtle_shell.rs file is missing :(

Reply:
that was odd, readded the files to the commit, it is saying there is a clean tree so there shouldn't be an issue now

Reply:
:+1: 
No worries, this happens quite often. I usually check "Files changed" tab here on GH while opening PR to double check that I am commiting the files I really want to be changed by the PR :)